### DISCUSSION TITLE
[Q] Can roadrunner work with other languages?

### DISCUSSION THREAD

**Discussion by easysoft** [roadrunner-server/roadrunner#697] [Q&A] [answered]
Hello,

Can roadrunner work with other languages a the same time? We developped an system mainly in PHP and our customer want to customize it by other language for example Java.

So is it possiable to combined PHP and Java at the same time by roadrunner?

Thanks a lot.

**Comment by rustatian** [comment, marked as answer]
Hey @easysoft. That's a great question. Sure, RR can work with other languages by redirecting HTTP, RPC, etc., requests to your JAVA application. I'll show that on todays' Twitch stream )

**Reply by rustatian** [reply]
Feel free to join our Discord server; I'll post the link somewhere around 3 PM

**Reply by rustatian** [reply]
Moreover, you can combine any languages with the Temporal plugin available at RoadRunner. You can write a worker in Java and send payloads to it from PHP or any desired language you want )
https://docs.temporal.io/docs/php/introduction/
https://docs.temporal.io/docs/java/introduction/

**Reply by easysoft** [reply]
Awesome, waiting the stream. Thanks a lot.
